extension/src/extension.ts

```diff
@@ -227,6 +238,19 @@ export class Extension extends Disposable implements IExtension {
 
     registerExperimentCommands(this.experiments, this.internalCommands)
     registerPlotsCommands(this.plots, this.internalCommands)
+    this.internalCommands.registerExternalCommand(
+      RegisteredCommands.EXPERIMENT_AND_PLOTS_SHOW,
+      async (context: VsCodeContext) => {
+        await this.experiments.showWebview(
```

Comment:
Does this need additional testing? Both of these show commands have testing already... I also wasn't sure where testing would even be located since it touches both plots and experiments.

Reply:
Maybe at least one to make sure both views are open.